### DISCUSSION TITLE
Transition from OpenIPC

### DISCUSSION THREAD

**Discussion by moosehadley** [themactep/thingino-firmware#329] [Q&A]
Hello, 

I have two Wyze cameras.  A Cam v2 and a Pan v1.
Both are already running OpenIPC.

Is there a simplified way to switch these cameras to Thingino?

The sysupgrade command in OpenIPC does not seem to work the same way it does in the Thingino documentation.
You have to specify --url to pull from the internet, and it expects a tgz archive.

**Comment by gtxaspec** [comment]
Hi,

We have included the `uniflasher` script in our repository to help with the transition. You can find it here:  
[uniflasher script](https://github.com/themactep/thingino-firmware/blob/master/scripts/uniflasher.sh)

To use it:
1. Download the script and firmware file for your device and save them to the `/tmp` directory (on the device).
2. Run the following command to flash the firmware:
   ```bash
   ./uniflasher /tmp/your_firmware_file.bin
   ```

Let us know if you need any further assistance!
